pyvista/core/pointset.py

```diff
@@ -56,6 +56,44 @@ def shallow_copy(self, to_copy):
             to_copy.SetPoints(vtk.vtkPoints())
         return Common.shallow_copy(self, to_copy)
 
+    def remove_cells(self, ind, inplace=True):
+        """Remove cells.
+
+        Parameters
+        ----------
+        ind : iterable
+            Cell indices to be removed.  The array can also be a
+            boolean array of the same size as the number of cells.
+
+        inplace : bool, optional
+            Updates mesh in-place while returning nothing when ``True``.
+
+        Examples
+        --------
+        Remove first 1000 cells from an unstructured grid.
+
+        >>> import pyvista
+        >>> letter_a = pyvista.examples.download_letter_a()
+        >>> letter_a.remove_cells(range(1000))
+        """
+        if isinstance(ind, np.ndarray) and ind.dtype == np.bool:
+            if ind.size != self.n_cells:
```

Comment:
This could now be a single if condition.

Reply:
Maybe I don't see it, but I think that it should remain a nested condition.  Having three conditions on one line is a bit messy.  In fact, for clarity, I'd almost prefer:
```python
        if isinstance(ind, np.ndarray):
            if ind.dtype == np.bool and ind.size != self.n_cells:
                raise ValueError('Boolean array size must match the '
                                 'number of cells (%d)' % self.n_cells)
```
vs:
```python
        if (isinstance(ind, np.ndarray) and ind.dtype == np.bool and
            ind.size != self.n_cells):
            raise ValueError('Boolean array size must match the '
                             'number of cells (%d)' % self.n_cells)
```

The reason for this is `size` and `dtype` are only attributes of `np.ndarray` and changing the order of the triple condition testing will cause an error.

Reply:
Resolved the parameterization in a5d40e51a61b470cff82ef4ef1ba4405578f4faa